packages/core/src/types/IDIDManager.ts

```diff
@@ -124,6 +124,28 @@ export interface IDIDManagerGetOrCreateArgs {
   options?: object
 }
 
+export interface IDIDManagerUpdateArgs {
+  /**
+   * Required. DID
+   */
+  did: string
+
+  /**
+   * Identifier alias. Can be used to reference an object in an external system
+   */
+  alias?: string
```

Comment:
Is the `alias` supposed to be updated, or is it being used to find some data related to this update?

I think it would be a good idea to separate local updates from remote (ledger) updates.
See also #948 for the proposal to update local metadata (`alias` would eventually fall under the metadata category, even if it is treated as a top level property now)

Reply:
The `alias` property can be skipped, as we are not passing the `provider` property, rather than relying on `did` as the predicate.
Initially, the rationale was to pass `alias` to be accessible to the provider. In that case, it can also be passed as metadata to the `options` object, which in turn is passed to the provider.
I will remove it from a top level arg.